lib/optimize.js

```diff
@@ -179,12 +179,14 @@ async function optimizeByType({ fileBuffer, filePath, isLossless }) {
             process.exit(1);
           });
 
+          child.on('close', () => {
+            resolve(Buffer.concat(buffers));
+          });
```

Comment:
> The 'close' event is emitted after a process has ended and the stdio streams of a child process have been closed. This is distinct from the 'exit' event, since multiple processes might share the same stdio streams. The 'close' event will always emit after 'exit' was already emitted, or 'error' if the child failed to spawn.

https://github.com/nodejs/node/blob/master/doc/api/child_process.md#event-close

> So, if you are only interested in the process termination (e.g. because the process holds an exclusive resource), listening for "exit" is sufficient. If you don't care about the program, and only about its input and/or output, use the "close" event.

https://stackoverflow.com/a/40898359